lib/pushmi_pullyu/aip/creator.rb

```diff
@@ -0,0 +1,69 @@
+require 'archive/tar/minitar'
+require 'bagit'
+require 'pushmi_pullyu/aip'
+
+class PushmiPullyu::AIP::Creator
+
+  attr_reader :aip_filename, :aip_directory
+
+  def initialize(noid)
+    @noid = noid
+    @aip_directory = "#{workdir}/#{@noid}"
+    @aip_filename = "#{workdir}/#{@noid}.tar"
```

Comment:
Couldn't you access these from the `PushmiPullyu::AIP` module?

Like `PushmiPullyu::AIP.aip_filename` etc? Or are these different? Seems to be some duplication